src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar-block calendar-block--start-day--sun calendar-block--month-length--29">
+
+      <div class="calendar-block__days"></div>
```

Comment:
Just as suggestion to simplify namings
```suggestion
    <div class="calendar calendar--start-day--sun calendar--month-length--29">

      <div class="calendar__day"></div>
```